### PR TITLE
Add new favicon

### PR DESCRIPTION
**Pull request by Cyanideankst**
refs TS-846

### CODE REVIEW

File: apps/sertra/pages/index.tsx [hunk 1]
```diff
@@ -27,7 +27,27 @@ const Home: NextPage<{ swrFallback: { [key: string]: unknown } }> = ({
...
+          />
+          <link
+            rel="icon"
+            type="image/png"
+            sizes="16x16"
+            href="/favicon-16x16.png"
+          />
+          <link rel="manifest" href="/site.webmanifest" />
+          <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5" />
+          <meta name="msapplication-TileColor" content="#da532c" />
```
Comment (MythicManiac) [inline apps/sertra/pages/index.tsx:49]: @gkurck can you verify these color choices are intended?

**Review by MythicManiac — APPROVED** [review]
Looks good otherwise, but let's wait for @gkurck to give input for the color choice 👍

File: apps/sertra/pages/index.tsx (hunk 1 above)
Comment (gkurck) [reply apps/sertra/pages/index.tsx:49]: I can verify they are **not** intended. The correct value for both is `#3d70c0`.

**Comment by Oksamies** [comment]
@MythicManiac Seems like the test pre-commit runner is getting stuck on something that at least my linter doesn't complain about.

**Review by Oksamies — APPROVED** [review]
All good and can merge, after the pre-commit runner has been looked at.

File: apps/sertra/public/browserconfig.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/mstile-150x150.png"/>
+            <TileColor>#da532c</TileColor>
```
Comment (gkurck) [inline apps/sertra/public/browserconfig.xml:6]: This one also has an incorrect value, correct value is `#3d70c0`.

**Review by gkurck — CHANGES_REQUESTED** [review]

**Review by gkurck — APPROVED** [review]

**Pull request merged**